### ISSUE TITLE
Add notice on npmjs.com that the timeago is no longer a node-compatible wrapper

### ISSUE THREAD

**Issue by jslatts** [rmm5t/jquery-timeago#231]
I was surprised to see that the timeago wrapper on npm I was using jumped from 0.2.X to 1.4.1 today when I went to update a project. 

I have no problem with the official timeago module taking over the npmjs name, but you really need to put some sort of warning on npm to indicate what has happened. I was super confused when I couldn't find @ecto's wrapper. Luckily some googling led me to issue #197 and I figured out what happened. 

It is especially important because without @ecto's wrapper, you can no longer require() and use it server side. Not sure if @ecto is going to re-publish the wrapper, but a link to that version (if published) would be helpful for those of us using it in node.

**Comment by rmm5t** [comment]
A PR is welcome here to remedy.

**Comment by jslatts** [comment]
To clarify for anyone else reading this, a PR should address making the module CommonJS compatible (http://blog.npmjs.org/post/112712169830/making-your-jquery-plugin-work-better-with-npm).

Not sure if/when I'll have time to do this. I have an acceptable work around in place for my current project at the moment.

**Comment by jslatts** [comment]
Turns out I needed to load timeago via browserify for use on our front-end, so I went ahead and did the CommonJS change. This will let you load it correctly as part of a browserify or webpack build. 

See PR #233.

You still cannot use timeago on the server due to a dependency on jquery.extend. One option would be to use a node module (underscore/lodash) for the extend call when loading server-side. Or just due the object merge manually since it just happens once.
